traffic_ops/traffic_ops_golang/deliveryservice/capacity.go

```diff
@@ -249,7 +249,7 @@ JOIN server as s ON s.profile = pr.id
 JOIN cdn as c ON c.id = s.cdn_id
 JOIN type as t ON s.type = t.id
 WHERE t.name LIKE 'EDGE%'
-AND pa.config_file = 'rascal-config.txt'
+AND pa.config_file = 'tm-config.txt'
```

Comment:
We can't just change what TO is querying for here when all the existing parameters likely use the `config_file` of `rascal-config.txt`. In general, we can either include a DB migration which will change all parameters from `rascal-config.txt` to `tm-config.txt`, make this code look for _either_ name, and or both of those.

Also, it would be good to use the constant `MonitorConfigFile` from the `monitoring` package here.

Reply:
Included migration file.